### PR TITLE
specify machine image in jaws deploy orb

### PR DESCRIPTION
**Pull request by Adam-McDevitt**

### CODE REVIEW

File: jaws-journey-deploy/orb.yml [hunk 1]
```diff
@@ -627,6 +627,7 @@ jobs:
     resource_class: << parameters.resourceClass >>
     machine:
       docker_layer_caching: false
+      image: ubuntu-2004:202201-02
```
Comment (ChrisFord-Ovo) [inline jaws-journey-deploy/orb.yml:630]: what is the purpose of this change?

Comment (ChrisFord-Ovo) [reply jaws-journey-deploy/orb.yml:630]: would it not be better to pass this in rather than hardcoding? i.e. << parameters.image >>

Comment (Adam-McDevitt) [reply jaws-journey-deploy/orb.yml:630]: yeh probably makes sense to just specify it once. I'll try something like << parameters.image >>

Comment (Adam-McDevitt) [reply jaws-journey-deploy/orb.yml]: @ChrisFord-Ovo pushed up some changes to stick this in a parameter

Comment (ChrisFord-Ovo) [reply jaws-journey-deploy/orb.yml]: cool - yeah also means consumers of the orb could change the image if they wanted - although not sure why

**Review by ChrisFord-Ovo — APPROVED** [review]
lgtm

**Review by ChrisFord-Ovo — APPROVED** [review]
wont you need to bump the version of the orb actually
